types/node/v13/tslint.json

```diff
@@ -3,6 +3,7 @@
   "rules": {
     "ban-types": false,
     "no-empty-interface": false,
+    "no-outside-dependencies": false,
```

Comment:
What's with these lint file changes?

Reply:
microsoft/dtslint#315 It's unrelated to this PR, the bot just won't cooperate unless the (GitHub Actions) CI [is passing](https://github.com/DefinitelyTyped/DefinitelyTyped/runs/1382252195#step:5:3166).